modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -22,20 +22,20 @@ def perform
       forms_data.dig('data', 'nodeQuery', 'entities').each do |form|
         build_and_save_form(form)
       rescue => e
-        log_message_to_sentry(
-          "#{form['fieldVaFormNumber']} failed to import into forms database", :error, body: e.message
-        )
+        message = "#{self.class.name} failed to import #{form['fieldVaFormNumber']} into forms database"
+        Rails.logger.error(message, e)
+        log_message_to_sentry(message, :error, body: e.message)
```

Comment:
Since I can't find any Sentry records for `VAForms` in Slack, I'm logging this error to the Rails console as well.